src/CommunalHelper.csproj

```diff
@@ -32,9 +32,6 @@
 		<Reference Include="$(CelestePrefix)\MMHOOK_Celeste.dll">
 			<Private>false</Private>
 		</Reference>
-		<Reference Include="$(CelestePrefix)\Celeste.dll">
```

Comment:
Not sure why this was removed? Would prefer to have any build changes done in separate PRs

Reply:
Because `Celeste.dll` is already included through the publicizer.
I dont't remember if it actually caused issue for me, but it's better to not include Celeste twice.

Reply:
Gotcha. I'll undo it in this PR and include it in a separate commit.